### ISSUE TITLE
Style of for loops enumeration example is unclear Ch. 4.6 of The Book.

### ISSUE THREAD

**Issue by jaredmanning** [rust-lang/rust#34624]
The example is as follows:
```
for (i,j) in (5..10).enumerate() {
    println!("i = {} and j = {}", i, j);
}
```
The first tuple does not have a space between the comma and the j -> `(i,j)`
Is this correct?

**Comment by xitep** [comment]
apart from the reported issue; in this example, it would be nice to name `j` something like `element` or just `v` (for "value") to make clear that `i` is the enumerated index. `i` and `j` are often used for index variables. maybe even falling back to longer names like "index" and "value" would be better.

in the mentioned book chapter, the example on using `lines.enumerate()` that follows the reported one, is very clear and self-explanatory.

**Comment by Blaisorblade** [comment]
The original issue appears addressed in #34625, which was merged (though it's not yet reflected in the manual, https://doc.rust-lang.org/book/loops.html).

There's still the comment of @xitep to address, and I agree with him.

**Comment by steveklabnik** [comment]
> (though it's not yet reflected in the manual, https://doc.rust-lang.org/book/loops.html).

Everything lands on nightly, then makes its way through the trains. So you can see it https://doc.rust-lang.org/nightly/book/loops.html , and it'll make it into 1.12's docs.
